spec/features/people/roles_form_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
```

Comment:
The name of this file is a bit off, since this isn't a form. Maybe just role_list_spec.rb?